pygameMenu/widgets/textinput.py

```diff
@@ -35,6 +35,27 @@
 from pygameMenu import locals as _locals
 from pygameMenu.widgets.widget import Widget
 
+try:
+    from pyperclip import copy, paste
+except ModuleNotFoundError:
```

Comment:
Does not exist, use `ImportError`

Reply:
Interesting, what version of python are you using? I tried to import any `import jjdkff..` and Python 3-64 thows ModuleNotFoundError. Could you add ImportError too? Maybe ImportError exist in both Python2/3

Reply:
`ModuleNotFoundError` is a subclass of `ImportError` introduced in Python3.6.
I think that the best is to use `ImportError` available for all Python versions.

Reply:
Aaah, much better then. Thanks! :yum: